### ISSUE TITLE
Delisting plugins

### ISSUE THREAD

**Issue by missmatsuko** [gridsome/gridsome.org#308]
Plugins seem to be automatically pulled into the plugin listing if they have `gridsome-plugin` keyword in `package.json`, but I found at least 1 plugin that isn't functional in the listing:
https://gridsome.org/plugins/@vroger/transformer-img

Is there some way to remove plugins from the list that aren't useful?
There should also be a way for devs to request being listed again, after they've made changes to their plugin.

**Comment by missmatsuko** [comment]
Here's some more that should probably be delisted, based on their README content:
https://gridsome.org/plugins/@108kb/gridsome-remark-plugin-gist
https://gridsome.org/plugins/transformer-yamljson
https://gridsome.org/plugins/gridsome-source-shopify

**Comment by tomtev** [comment]
Yes we should make a file where we can blacklist plugins that will filtered out. I can try create something.
